### PR TITLE
Move noSDK into config

### PR DESCRIPTION
**Pull request by jeffsee55**
This shows a warning if `--noSDK` is provided, but that property still works for now. 

```js
{
  client: {
    skip: true
  }
}
```

The work to add this to part of the forestry migration init hasn't been done yet.

Closes ENG-869

### CODE REVIEW

File: packages/@tinacms/app/src/Playground.tsx [hunk 1]
```diff
@@ -73,7 +73,7 @@ const Playground = () => {
         <div className="graphiql-doc-explorer-content">
           <div className="graphiql-markdown-description">
             {noAutoQueries
-              ? 'No autoqueries found, the Tina dev or build command was likely run with the "--noSDK" option. Learn about how to generate autoqeries.'
+              ? 'No autoqueries found, the Tina config is likey set to client.skip = true. Learn about how to generate autoqeries.'
```
Comment (jamespohalloran) [inline packages/@tinacms/app/src/Playground.tsx:76]:
```suggestion
              ? 'No autoqueries found. The Tina config is likely set to "client.skip = true". Learn about how to generate autoqueries.'
```
^ Not new to this, but kind of seems like we meant to include a link at the end of that. I also don't think we refer to "autoqueries" in our docs.

File: packages/@tinacms/cli/src/next/commands/build-command/index.ts [hunk 2]
```diff
@@ -29,7 +29,7 @@ export class BuildCommand extends Command {
     description: 'increase verbosity of logged output',
   })
   noSDK = Option.Boolean('--noSDK', false, {
-    description: "Don't generate the generated client SDK",
+    description: "DEPRECATED - Don't generate the generated client SDK",
```
Comment (jamespohalloran) [inline packages/@tinacms/cli/src/next/commands/build-command/index.ts:32]: Might be good to mention to new solution here, or link to a doc

**Review by jamespohalloran — CHANGES_REQUESTED** [review]

File: packages/@tinacms/app/src/Playground.tsx (hunk 1 above)
Comment (jeffsee55) [reply packages/@tinacms/app/src/Playground.tsx]: Will follow up with a link in another PR. There are some other playground improvements needed.

File: packages/@tinacms/cli/src/next/commands/build-command/index.ts (hunk 2 above)
Comment (jamespohalloran) [reply packages/@tinacms/cli/src/next/commands/build-command/index.ts:32]: I don't see why not

Comment (jeffsee55) [inline packages/@tinacms/cli/src/next/commands/build-command/index.ts:32]:
```suggestion
    description: "DEPRECATED - This should now be set in the config at config.skip = true'". Don't generate the generated client SDK",
```

File: packages/@tinacms/cli/src/next/commands/dev-command/index.ts [hunk 3]
```diff
@@ -46,7 +46,7 @@ export class DevCommand extends Command {
     description: "Don't regenerate config on file changes",
   })
   noSDK = Option.Boolean('--noSDK', false, {
-    description: "Don't generate the generated client SDK",
+    description: "DEPRECATED - Don't generate the generated client SDK",
```
Comment (jeffsee55) [inline packages/@tinacms/cli/src/next/commands/dev-command/index.ts:49]:
```suggestion
    description: "DEPRECATED - This should now be set in the config at config.skip = true'. Don't generate the generated client SDK",
```

**Review by logan-anderson — APPROVED** [review]

**Pull request merged**